app/models/doctor_establishment.rb

```diff
@@ -0,0 +1,4 @@
+class DoctorEstablishment < ApplicationRecord
+  belongs_to :doctor_id
+  belongs_to :establishment_id
```

Comment:
@aniketpatidar Lets meet and discuss how you are creating these migrations because its a bit incorrect but if done the right way you should not get doctor_id and establishemt_id here. 
If you don't have time to meet you can check this video too: https://youtu.be/xNQYsUS8Tcs